downstream/modules/installing_ocp_prerequisites.adoc

```diff
@@ -75,6 +77,8 @@ See https://access.redhat.com/articles/3174981[How do I Download and Install Red
 + 
 See https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands[Installing the CLI] in the _OpenShift documentation_.
```

Comment:
```suggestion
See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/cli_tools/openshift-cli-oc#installing-the-cli[Installing the CLI] in the _OpenShift documentation_.
```

Under "Reference" https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/ you will find "CLI Tools"

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/cli_tools/index